tests/frequentist/test_ztest.py

```diff
@@ -3450,6 +3450,51 @@ def test_multiple_difference_groupby(self):
         np.testing.assert_almost_equal(difference_df[DIFFERENCE].values[0], -0.0149, 3)
 
 
+class TestSequentialOneSidedThreeGroups(object):
+    def setup(self):
+        DATE = "date"
+        COUNT = "count"
+        SUM = "sum"
+        SUM_OF_SQUARES = "sum_of_squares"
+        GROUP = "group"
+
+        self.data = pd.DataFrame(
+            [
+                {DATE: 1, GROUP: "1", COUNT: 500, SUM: 1000, SUM_OF_SQUARES: 3000},
+                {DATE: 1, GROUP: "2", COUNT: 1000, SUM: 1000, SUM_OF_SQUARES: 6000},
+                {DATE: 1, GROUP: "3", COUNT: 1500, SUM: 1000, SUM_OF_SQUARES: 12000},
+                {DATE: 2, GROUP: "1", COUNT: 1000, SUM: 2000, SUM_OF_SQUARES: 6000},
+                {DATE: 2, GROUP: "2", COUNT: 2000, SUM: 2000, SUM_OF_SQUARES: 12000},
+                {DATE: 2, GROUP: "3", COUNT: 3000, SUM: 2000, SUM_OF_SQUARES: 24000},
+            ]
+        ).assign(final_expected_sample_size=1e4)
+
+        self.test = spotify_konfidens.ZTest(
```

Comment:
spotify_konfidens, is that defined somewhere?

Reply:
Oops, typo!

Reply:
Did you run tests locally? It's a good idea to run tests, e.g. using tox, and to also run `make black` and then flake8 locally to make sure formatting etc is correct, because otherwise our github workflows will fail anyways.

Reply:
Can't run this repo locally atm because of incompatibility with mac M1, but ran the linter + black. (We have an internal fork where M1 issues has been fixed where I ran all the tests, but haven't contributed it back yet)

Reply:
Works on my M1 max laptop. 🙂